Akretion/9.0 migration account fiscal rule

mbcosta opened the pull request.

mbcosta committed:
Migration the models, add new dependecy and set module installable.

mbcosta committed:
Migration the views and add data file ( because they were removed from account module ).

mbcosta committed:
Migration the test file.

mbcosta committed:
Attribute description removed from a class.

mbcosta committed:
Include description of module and change encoding to conding.

danimaribeiro commented on account_fiscal_position_rule/views/account_fiscal_position_rule_view.xml:
```diff
@@ -1,220 +1,231 @@
+			<field name="arch" type="xml">
+				<form string="Fiscal Position Rule Template">
+					<group>
+						<group string="General" colspan="4">
```
I suggest put the name attribute in the groups, in version 9 we can't use the string attribute to inherit views anymore.

mbcosta replied:
Hi danimaribeiro, thanks for the feedbak, I add the attribute name

mbcosta committed:
[MIG] Add attribute name to allow inherit the view.

rvalyi commented:
@mbcosta: just a hint in case you don't know it: when you mention somebody in a comment, if you use @ like @danimaribeiro then you are sure that person will be notified. Else it depends a bit if the person is following the thread or not. The fix makes sense BTW.

mbcosta commented:
Thanks for the hint @rvalyi

mbcosta committed:
Change field Fiscal Position to be required. (#53)

[IMP] account_fiscal_position_rule: Change field Fiscal Position to be required

rvalyi commented on account_fiscal_position_rule/models/account_fiscal_position_rule.py:
```diff
@@ -116,52 +115,47 @@ def _map_domain(self, partner, addrs, company, **kwargs):
-    def apply_fiscal_mapping(self, result, **kwargs):
-        result['value'].update(self.fiscal_position_map(**kwargs))
+    def apply_fiscal_mapping(self, **kwargs):
+        result = self.fiscal_position_map(**kwargs)
```
better to write simply return self.fiscal_position_map(**kwargs)

rvalyi replied:
please use simply return self.fiscal_position_map(**kwargs)

mbcosta committed:
Refactoring

mbcosta committed:
Remove comment.

rvalyi commented:
:+1:

renatonlima commented:
:+1:
